scalableminds/webknossos
# Slow down brush size change for small sizes

This PR changes the change of the brush size from a constant value to one that adjusts to the current size. When the size is small the changes are smaller than before. And when the size passes a certain threshold the value of change is kept to a maximum of 5.
### URL of deployed dev instance (used for testing):
- [https://logarithmicbrushsizechangeforkeyboard.webknossos.xyz](https://logarithmicbrushsizechangeforkeyboard.webknossos.xyz)

### Steps to test:
- Open a volume or hybrid tracing.
- Select the brush.
- Change the brush size with `shift + mouse wheel` and `shift + i/o`.
- The size should change slowly when having a small size and the change should increase as the size increases. 

### Issues:
- fixes #3964 

------
